$OpenBSD: patch-exec_make_profile_in,v 1.4 2012/01/14 22:14:10 dcoppa Exp $

remove bashism
fix default config file path
use some sane defaults for menu/launchbar apps: that's a best effort 
task :-)

--- exec/make_profile.in.orig	Wed Mar 24 10:58:10 2010
+++ exec/make_profile.in	Fri Jan 13 18:11:16 2012
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 umask 0077
 # new profile directory
@@ -6,7 +6,7 @@ npdir=~/.config/fbpanel
 # old profile directory
 opdir=~/.fbpanel
 # system profile directory
-spdir=%%datadir%%/fbpanel
+spdir=%%datadir%%/examples/fbpanel
 # if profile name was not set, use  'default'
 profile=${1:-default}
 
@@ -42,24 +42,24 @@ fi
 # Creates new profile using system profile as template
 # $1 - system profile name
 # $2 - destination profile name
-function take_system_profile ()
+take_system_profile ()
 {
     [ -r "$spdir/$1" ] || return 1
 
     local browser terminal filer
-    for browser in x-www-browser firefox opera; do
+    for browser in firefox firefox36 firefox35 chrome midori xxxterm epiphany opera; do
         if which $browser 2> /dev/null > /dev/null; then
             opt="$opt -e s/x-www-browser/$browser/"
             break
         fi
     done
-    for terminal in x-terminal urxvt gnome-terminal; do
+    for terminal in urxvt gnome-terminal xfce4-terminal konsole roxterm xterm; do
         if which $terminal 2> /dev/null > /dev/null; then
             opt="$opt -e s/x-terminal/$terminal/"
             break
         fi
     done
-    for filer in x-file-manager thunar pcmanfm rox; do
+    for filer in Thunar nautilus pcmanfm emelfm2 rox krusader konqueror; do
         if which $filer 2> /dev/null > /dev/null; then
             opt="$opt -e s/x-file-manager/$filer/"
             break
